Project-II-Calculator/calculator/src/App.js

```diff
@@ -11,21 +11,21 @@ const App = () => {
     <div className='container'>
       <CalculatorDisplay />
       <ActionButton buttonStyle='clear' text='clear'/>
-      <NumberButton buttonStyle='red' text='/'/>
-      <NumberButton buttonStyle='white' text='7'/>
-      <NumberButton buttonStyle='white' text='8'/>
-      <NumberButton buttonStyle='white' text='9'/>
-      <NumberButton buttonStyle='red' text='X'/>
-      <NumberButton buttonStyle='white' text='4'/>
-      <NumberButton buttonStyle='white' text='5'/>
-      <NumberButton buttonStyle='white' text='6'/>
-      <NumberButton buttonStyle='red' text='-'/>
-      <NumberButton buttonStyle='white' text='1'/>
-      <NumberButton buttonStyle='white' text='2'/>
-      <NumberButton buttonStyle='white' text='3'/>
-      <NumberButton buttonStyle='red' text='+'/>
+      <NumberButton buttonStyle='reds' text='÷'/>
```

Comment:
This is not very DRY. Think about what it would take to be able to implement a map for your buttons. Would you benefit from separating the whites and reds into different components?